client/web/compose/src/components/PageBlocks/Navigation/Base.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <wrap
+    v-bind="$props"
+    v-on="$listeners"
+  >
+    <div class="h-100 w-100 navigation-page-block-item">
```

Comment:
replace navigation-page-block-item with actual bootstrap classes. Remember we want to avoid new classes as much as possible

Reply:
the reason I made use of the class is because I could not easily edit the styling on the nav list item to take the full height of the block, and I did not want to make the stlyes get affected by something else i the app, `scoped` is not helping because its a component
<img width="345" alt="image" src="https://user-images.githubusercontent.com/23015247/209345710-e87c1824-be19-4c8c-86b2-b12c1ea84c64.png">

Reply:
Not sure what you mean, you can control the class where you call the b-nav-item with the class property

Reply:
I have moved the class to the `b-nav` component